examples/calendar/src/components/Calendar.tsx

```diff
@@ -65,30 +54,30 @@ const isClickOnCalendarEvent = (xy: MouseCoordinates) =>
     isElAtPoint(xy, StylesCalendarEvent.event);
 
 type Props = {
-    DEBUG_ORDER: boolean,
-    displayMonth: Date,
-    events: Array<EventRecord>,
-    focusedEvent: EventRecord,
-    isMobileApp: boolean,
-    isSmallScreen: boolean,
-    menuOpenRecord: EventRecord,
-    mouseCoordinates: MouseCoordinates,
-    mouseStartCoordinates: MouseStartCoordinates,
-    movingEvent: ?EventRecord,
-    movingEventOrder: ?MovingEventOrder,
-    refreshEvents: Function,
-    resizingEvent: ?EventRecord,
-    rootRecord: RootRecord,
-    selectedEvent: EventRecord,
-    setDebugProp: Function,
-    setFocusedEvent: Function,
-    setMouseCoordinates: Function,
-    setMovingEvent: Function,
-    setMovingEventOrder: Function,
-    setMovingEventRectMap: Function,
-    setIsSmallScreen: Function,
-    setResizingEvent: Function,
-    setSelectedEvent: Function,
+    DEBUG_ORDER: boolean;
+    displayMonth: Date;
+    events: Array<EventRecord>;
+    focusedEvent: EventRecord;
+    isMobileApp: boolean;
+    isSmallScreen: boolean;
+    menuOpenRecord: EventRecord;
+    mouseCoordinates: MouseCoordinates;
+    mouseStartCoordinates: MouseStartCoordinates;
+    movingEvent: EventRecord | undefined | null;
```

Comment:
nit: we generally do `movingEvent?: EventRecord | null` to annotate optional keys rather than using undefined. Note that this is irrelevant to this PR, just something I noticed when scanning.